changelog/march2024/2024-03-20-jobs-added-custom-ephemeral-storage-on-serverless-jobs.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Custom Ephemeral Storage on Serverless Jobs
+status: added
+author:
+    fullname: 'Join the #serverless-jobs-beta channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-20
+category: serverless
+product: jobs
+---
+
+Serverless Jobs now offers the possibility to customise your custom ephemeral storage! 
+
+Each Serverless Job has a limited storage a now you can customise it to allow more demanding use cases.
```

Comment:
```suggestion
With custom ephemeral storage, your Serverless Jobs no longer have a limited storage, as you can customize them to allow more demanding use cases.
```

Reply:
Ephemeral storage is still limited to 10GB (used to be 5), so maybe
```suggestion
Each Serverless Job comes with an ephemeral storage capacity that you can now customize to accommodate for more demanding use cases.
```

Reply:
cc @nerda-codes